pycode/memilio-epidata/memilio/epidata/modifyDataframeSeries.py

```diff
@@ -188,7 +188,7 @@ def extract_subframe_based_on_dates(df, start_date, end_date, moving_average = 0
 
         return df
 
-    except DataError:
+    except:
```

Comment:
We do not want to have unqualified excepts in this module. I think its better here to not catch any error and just let the program raise the original error message.